utils/dwarf.h

```diff
@@ -34,8 +34,9 @@ struct uftrace_dbg_loc {
 };
 
 struct uftrace_dbg_info {
-	/* opaque DWARF info pointer */
-	Dwarf *dw;
+	/* opaque DWARF frontend library pointers */
+	Dwfl *dwfl;
+	Dwfl_Module *dwfl_module;
```

Comment:
Note that `struct uftrace_dbg_info` is used even when `HAVE_LIBDW` is NOT defined.  You'd better defined the data types as `void` like above.